quart_2/seminar_7/my_phonebook/csv_action.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+path = 'phonebook.csv'
+
+
+def counter_lines():
+    count = 0
+    with open(path, 'r', encoding='utf-8') as f:
+        for line in f:
+            count += 1
+    return str(count)
```

Comment:
Цикл не обязателен, можно сделать так.
```suggestion
        return str(len(f.readlines()))
```